config/initializers/content_security_policy.rb

```diff
@@ -27,6 +30,7 @@
     policy.style_src(*self_base)
     policy.connect_src(*self_base.concat(ga_connect_src))
     policy.frame_src(*self_base.concat(gtm_frame_src))
+    policy.style_src_elem(*self_base.concat(["'unsafe-inline'"]))
```

Comment:
I think its worth looking into what's causing this; we want to avoid `unsafe-inline` if at all possible

Reply:
discussed in person: we are going to add this to the only offending place which is in an appropriate body view, and add it as a new concern specific to that controller and not disable script source everywhere, style should be ok. We also agreed to keep report only for now, and merge the pr, and then 2 days later we'll remove report only